### PR TITLE
DescribeCommand - filter code samples and output note when none can be demonstrated

### PR DESCRIPTION
**Pull request by localheinz**
This PR

* [x] filters code samples and outputs a note when none of them can be demonstrated on the current PHP version

Follows #2354.
Follows https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/2351#discussion_r91832703.

### CODE REVIEW

File: src/Console/Command/DescribeCommand.php [hunk 1]
```diff
@@ -139,7 +140,17 @@ private function describeRule(OutputInterface $output, $name)
...
+        $codeSamples = array_filter($definition->getCodeSamples(), function (CodeSampleInterface $codeSample) {
+            if ($codeSample instanceof VersionSpecificCodeSampleInterface) {
+                return $codeSample->isSuitableFor(PHP_VERSION_ID);
+            }
+
+            return true;
+        });
+
+        if (!count($codeSamples)) {
+            $output->writeln('Fixing examples can not be demonstrated on the current PHP version.');
```
Comment (localheinz) [inline src/Console/Command/DescribeCommand.php]: Shall I use some formatting here, if so, which one would you like?

**Comment by localheinz** [comment]
Thank you, @keradus!

**Comment by SpacePossum** [comment]
:+1: nice :)

**Comment by GrahamCampbell** [comment]
:+1: 🍻

**Comment by keradus** [comment]
Thank you @localheinz.

**Pull request merged**